src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -764,29 +766,46 @@ class FlaxBertPreTrainedModel(FlaxPreTrainedModel):
 
     config_class = BertConfig
     base_model_prefix = "bert"
-    module_class: nn.Module = None
+    if typing.TYPE_CHECKING:
```

Comment:
Again maybe cleaner to stick with `module_class = None` and address the ClassVar issue in a follow-up (/parallel) PR, WDYT?

Reply:
Sounds good, will remove all the `TYPE_CHECKING` usage